app/src/main/java/org/openobservatory/ooniprobe/common/DateAdapter.java

```diff
@@ -0,0 +1,37 @@
+package org.openobservatory.ooniprobe.common;
```

Comment:
Can you submit just this `DateJsonSerializer` => `DateAdapter` as a separate change?